src/main/webapp/web.xml

```diff
@@ -3,12 +3,12 @@
          xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee
                       http://xmlns.jcp.org/xml/ns/javaee/web-app_4_0.xsd"
          version="4.0">
-    <servlet>
-        <servlet-name>index</servlet-name>
-        <servlet-class>mate.controller.IndexController</servlet-class>
-    </servlet>
-    <servlet-mapping>
-        <servlet-name>index</servlet-name>
-        <url-pattern>/index</url-pattern>
-    </servlet-mapping>
+<!--    <servlet>-->
+<!--        <servlet-name>index</servlet-name>-->
+<!--        <servlet-class>mate.controller.IndexController</servlet-class>-->
+<!--    </servlet>-->
+<!--    <servlet-mapping>-->
+<!--        <servlet-name>index</servlet-name>-->
+<!--        <url-pattern>/index</url-pattern>-->
+<!--    </servlet-mapping>-->
```

Comment:
Let's remove completely the mapping, don't leave code comments in your solution.